Core/App/AppInstaller.php

```diff
@@ -324,10 +325,8 @@ private function testMysql($dbData)
         }
 
         // Omit the DB name because it will be checked on a later stage
-        $connection = new \mysqli($dbData['host'], $dbData['user'], $dbData['pass'], '', (int) $dbData['port']);
-        if ($connection->connect_error) {
-            $this->miniLog->critical((string) $connection->connect_error);
-        } else {
+        $connection = @new \mysqli($dbData['host'], $dbData['user'], $dbData['pass'], '', (int) $dbData['port']);
```

Comment:
Avoid that connection throw a message, because we are using the same message to notify the end user in case of any error happens.